src/rust_version.rs

```diff
@@ -10,17 +10,17 @@ enum FetchTarget {
     ReleasesFeed,
 }
 
-fn fetch(target: FetchTarget) -> Result<reqwest::Response, Box<dyn Error>> {
+fn fetch(target: FetchTarget) -> Result<reqwest::blocking::Response, Box<dyn Error>> {
     let proxy_env = env::var("http_proxy")
         .or_else(|_| env::var("HTTPS_PROXY"))
         .ok();
 
     let client = match proxy_env {
         Some(proxy_env) => {
             let proxy = reqwest::Proxy::https(&proxy_env).unwrap();
-            reqwest::ClientBuilder::new().proxy(proxy).build().unwrap()
+            reqwest::blocking::ClientBuilder::new().proxy(proxy).build().unwrap()
```

Comment:
It's this line that needs to be formatted.

Reply:
Diff in /home/runner/work/www.rust-lang.org/www.rust-lang.org/src/rust_version.rs at line 18:
     let client = match proxy_env {
         Some(proxy_env) => {
             let proxy = reqwest::Proxy::https(&proxy_env).unwrap();
-            reqwest::blocking::ClientBuilder::new().proxy(proxy).build().unwrap()
+            reqwest::blocking::ClientBuilder::new()
+                .proxy(proxy)
+                .build()
+                .unwrap()
         }
         None => reqwest::blocking::Client::new(),
     };
##[error]Process completed with exit code 1.

I'm unsure.  Above shows the call broken up over multiple lines, but my source doesn't show this.

Reply:
Yes, I believe it wants you to format it like this.
```suggestion
            reqwest::blocking::ClientBuilder::new()
                .proxy(proxy)
                .build()
                .unwrap()
```

Reply:
Oh, ok!